Dockerfile

```diff
@@ -4,20 +4,45 @@ FROM public.ecr.aws/docker/library/fedora:37 as base
 RUN \
   dnf makecache && \
   dnf -y update && \
-  dnf -y groupinstall "C Development Tools and Libraries" && \
   dnf -y install --setopt=install_weak_deps=False \
-    rpmdevtools dnf-plugins-core createrepo_c \
-    cmake git meson perl-ExtUtils-MakeMaker python which \
-    bc hostname intltool gperf kmod rsync wget openssl \
-    dwarves elfutils-devel libcap-devel openssl-devel \
-    createrepo_c e2fsprogs gdisk python3-jinja2 \
-    kpartx lz4 veritysetup dosfstools mtools squashfs-tools \
-    perl-FindBin perl-IPC-Cmd perl-open policycoreutils \
-    secilc qemu-img glib2-devel rpcgen erofs-utils jq ShellCheck \
-    json-c-devel libcurl-devel p11-kit-devel && \
+    bc \
+    bison \
+    cmake \
+    cpio \
+    curl \
+    dnf-plugins-core \
+    dwarves \
+    elfutils-devel \
+    flex \
+    g++ \
+    gcc \
+    git \
+    gperf \
+    hostname \
+    intltool \
+    jq \
+    json-c-devel \
+    kmod \
+    libcurl-devel \
+    libtool \
+    meson \
+    openssl \
+    openssl-devel \
+    p11-kit-devel \
+    perl-ExtUtils-MakeMaker \
+    perl-FindBin \
+    perl-IPC-Cmd \
+    perl-open \
+    python \
+    rsync \
+    wget \
+    which \
+  && \
   dnf config-manager --set-disabled \
-    fedora-modular updates-modular fedora-cisco-openh264 && \
-  dnf clean all && \
+    fedora-modular \
+    updates-modular \
+    fedora-cisco-openh264 \
```

Comment:
Nit: these commands are reformatted here, but were added in the first commit of the series. Probably the diff will be smaller if the format doesn't change(?).

Reply:
I think they should have been this newer format originally, and the sooner we change it then the sooner it's in better shape for ongoing updates where the diff can show individual lines more clearly.

Reply:
Right, so probably keep this format in the first commit of the series

Reply:
This gets back to whether we treat our unit of change as individual commits or the pull request. I personally am not as concerned about individual commits in a PR unless there is a chance one of those commits may need to be reverted at some point.

Reply:
I'm not sure it would be worth the hassle.

Reply:
Fair point - I'd tidy this up if there were another change, but I don't think it's worth the retesting burden just for commit history. Style-wise, that was consistent with the rest of `Dockerfile` at the time the change was made, so it's not indefensible. 😀